### PR TITLE
dodanie pocisku, w godocie ustawienie prymitywnego hitboxa

### PR DESCRIPTION
**Pull request by Githubfun001**

### CODE REVIEW

File: Player/player.cs [hunk 1]
```diff
@@ -3,20 +3,22 @@
...
 	private Texture2D backTexture;
+	PackedScene bulletScene;
 
 
 	public override void _Ready()
 	{
 		postac = GetNode<Sprite2D>("Sprite2D");
 		frontTexture = (Texture2D)GD.Load("res://Textures/Player/Postac.png");
 		backTexture = (Texture2D)GD.Load("res://Textures/Player/Postac_tyl.png");
+		bulletScene = GD.Load<PackedScene>("res://Player/bullet.tscn");
```
Comment (petriNicholas) [inline Player/player.cs:21]: in the future when we will have many loads, we need to shorten this string-links by having consts 

but we will need it in the future so keep it in mind
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants

File: Player/player.cs [hunk 2]
```diff
@@ -31,7 +33,7 @@ public override void _PhysicsProcess(double delta)
 		isMovingUp = moveVector.Y < 0;
 		}
 		// Wzór na szybkość ruchu
-		Vector2 velocity = moveVector.Normalized() * (float)speed * (float)delta;
+		Vector2 velocity = moveVector.Normalized() * speed * (float)delta;
```
Comment (petriNicholas) [inline Player/player.cs:36]: if we are taking delta as a double as a parameter, why are we changing it in this line to float?

maybe we can just take delta parameter as a double

File: Player/player.cs [hunk 3]
```diff
@@ -47,4 +49,25 @@ public override void _PhysicsProcess(double delta)
...
 		}
 	}
+
+	public override void _UnhandledInput(InputEvent @event)
+	{
+		if (@event is InputEventMouseButton mouseEvent)
+		{
+			if (mouseEvent.ButtonIndex == MouseButton.Left && mouseEvent.Pressed)
+			{
+				bullet bullet = (bullet)bulletScene.Instantiate();
```
Comment (petriNicholas) [inline Player/player.cs:59]: class' name should start with capital letter

**Review by petriNicholas — CHANGES_REQUESTED** [review]
change the class name to capital
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines

File: Player/player.cs (hunk 2 above)
Comment (Githubfun001) [reply Player/player.cs:36]: you cannot multiply float with double

**Review by petriNicholas — APPROVED** [review]

**Pull request merged**